king_phisher/client/assistants/campaign.py

```diff
@@ -127,6 +172,37 @@ def __init__(self, application, campaign_id=None):
 			self.gobjects['label_intro_body'].set_text('This assistant will walk you through creating and configuring a new King Phisher campaign.')
 			self.gobjects['label_intro_title'].set_text('New Campaign')
 
+		self._url_selected = None
+		self._url_classifiers = {}
+		self._url_thread = None
+
+		self.domain_completion = Gtk.EntryCompletion()
+		self._domain_list_store = Gtk.ListStore(str)
+		self.domain_completion.set_model(self._domain_list_store)
+		self.domain_completion.set_text_column(0)
+		self.gobjects['entry_domain_filter'].set_completion(self.domain_completion)
+
+		self._import_kpm = KpmImport()
+
+		tvm = managers.TreeViewManager(self.gobjects['treeview_url_selector'])
+		tvm.set_column_titles(
+			['Domain', 'Landing Page', 'URL', 'Classifiers'],
+			renderers=[
+				Gtk.CellRendererText(),
+				Gtk.CellRendererText(),
+				Gtk.CellRendererText(),
+				Gtk.CellRendererText(),
+			]
+		)
+		self._url_model = Gtk.TreeStore(str, str, str, str, str, str, str)
+		self._url_model.set_sort_column_id(0, Gtk.SortType.ASCENDING)
+		self.gobjects['treeview_url_selector'].set_model(self._url_model)
+		self._url_information = {
```

Comment:
You're adding 5 `_url_*` attributes. Could you combine some into this `_url_information` dictionary, `selected` and `classifiers` both seem like they'd fit. You'd then have the raw data, the created timestamp, the selection and the classifiers (which came from the raw data) all together.